operations/multinode/src/main/application/schemas/music.sd

```diff
@@ -0,0 +1,25 @@
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

Comment:
Do we need to duplicate the schema file or can we use the schema file associated with the feed files: @../../album-recommendation-selfhosted/?

Reply:
good point, will symlink instead

Reply:
otoh, it is not duplicated but rather simplified (less rank profiles), so to have less deploy warnings etc will keep the schema